run/GCHP/setCommonRunSettings.sh.template

```diff
@@ -254,7 +285,16 @@ if [[ ${TomasDustEntry} != "missing" ]]; then
             echo "WARNING: Online dust (TOMAS_DustDead) is not recommended for stretched-grid simulation. Consider using OFFLINE_DUST."
         fi
         if [[ $CS_RES -eq 24 ]]; then
-            TomasDust_SF=6.0e-4
+	    if [[ "x${metField}" == "xMERRA2" ]]; then
```

Comment:
I wonder if we could just combine this with the earlier one.

Reply:
@lizziel: We have simplified the logic where mass tuning factors are set in commit d08e541762be53774d818deaf691a59df4890c47.